contracts/eigenlayer/script/WavsDelegateToOperator.s.sol

```diff
@@ -0,0 +1,87 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import {Script} from "forge-std/Script.sol";
+import {IDelegationManager} from "@eigenlayer/contracts/interfaces/IDelegationManager.sol";
+import {ISignatureUtilsMixinTypes} from "@eigenlayer/contracts/interfaces/ISignatureUtilsMixin.sol";
+
+import {WavsServiceManager} from "../src/WavsServiceManager.sol";
+
+contract WavsDelegateToOperator is Script {
```

Comment:
I didn't do a deep dive here, but it seems like one step.

When we register an operator, we need to ensure there is eth, then send that to the liquid staking protocol. then deposit that in eigenlayer, then self-delegate to the operator.

I was expected a similar flow here (which could pull a lot of the code from the register_operator. Maybe even refactor that and just call into it to do a few steps.